src/components/Table/Internal/Body/Scroller.tsx

```diff
@@ -185,7 +191,10 @@ export const Scroller = React.forwardRef(
         <SecondaryButton
```

Comment:
this is the side scroller button that is used in TA tables?

Reply:
Yes.